interop-cats/shared/src/main/scala/zio/interop/cats.scala

```diff
@@ -385,3 +387,18 @@ final private class CatsContravariant[E, T] extends Contravariant[ZIO[*, E, T]]
   override def contramap[A, B](fa: ZIO[A, E, T])(f: B => A): ZIO[B, E, T] =
     ZIO.accessM[B](b => fa.provide(f(b)))
 }
+
+final private class NonEmptyChunkReducible extends Reducible[NonEmptyChunk] {
+
+  override def foldLeft[A, B](fa: NonEmptyChunk[A], b: B)(f: (B, A) => B): B = fa.foldLeft(b)(f)
+  override def foldRight[A, B](fa: NonEmptyChunk[A], lb: Eval[B])(f: (A, Eval[B]) => Eval[B]): Eval[B] =
+    fa.foldRight(lb)(f)
+
+  override def reduceLeftTo[A, B](fa: NonEmptyChunk[A])(f: A => B)(g: (B, A) => B): B =
+    fa.tail.foldLeft(f(fa.head))(g)
+
+  override def reduceRightTo[A, B](fa: NonEmptyChunk[A])(f: A => B)(g: (A, Eval[B]) => Eval[B]): Eval[B] =
```

Comment:
Why is `last` unsafe? The `NonEmptyChunk` is guaranteed to be non-empty so there should always be a last element.